config/install/media_webdam.webdamupload.yml

```diff
@@ -0,0 +1 @@
+media_webdam:
```

Comment:
Can you remove this file entirely?

Reply:
oops! can I add config/ dir to the .gitignore?

Reply:
No, just delete the file. We will eventually have other config that we need to add here.